### DISCUSSION TITLE
Zusammenfassen von Lastschriften für verschiedene Mitglieder mit gleicher Kontoverbindung

### DISCUSSION THREAD

**Discussion by dominformant** [openjverein/jverein#303] [Ideas]
Beim Lesen der Diskussion zu #286 gefiel mir die anfängliche Idee, Mitgliedern die Abbuchung der Beiträge über ein anderes Mitglied zu erlauben. Wir haben im Verein mehrere Familien, bei denen die Beiträge für mehrere Mitglieder über die gleiche Kontoverbindung laufen. Der Abrechnungslauf erzeugt in diesem Fall allerdings für jedes Mitglied eine eigene Lastschrift, was zu (wenn auch niedrigen) unnötigen Kontoführungsgebühren führt. Schön wäre es, wenn alle Lastschriften für eine Kontoverbindung zu einer Buchung zusammengefasst werden könnten. Noch schöner wäre natürlich, auch das SEPA-Mandat nur einmal eingeben zu müssen und bei den übrigen Familienmitgliedern nur noch zu referenzieren. Über Familienmitgliedschaften, so wie sie bisher umgesetzt sind und, wenn ich es richtig verstanden habe, auch nach #286 lässt sich dies leider nicht schön umsetzen, da dann unzählige Beitragsgruppen für alle möglichen Kombinationen von Beiträgen erstellt werden müssten, bzw. Zusatzbeiträge immer noch separat abgebucht würden.

**Comment by lenilsas** [comment]
Ja, ich dachte erst dass ich es so umsetze, dass die Beiträge über den Zahler bezahlt werden. Hier war jedoch die Meinung, dass man ja bei abweichendem Kontoinhaber die Daten des Zahlers eintragen kann. Bei vielen Familien im Verein würde es vielleicht wirklich Sinn machen das noch zu ändern. Eine Möglichkeit wäre, dass man bei Angehörigen unter "Zahlungsweg" eine neue Option "durch Vollzahler" hinzufügt.
Der Sinn des Familienverbandes ist es in erster Linie ermäßigte Beitragsgruppen für Angehörige zu erstellen, so dass sichergestellt ist, dass immer ein Vollzahler pro Familie vorhanden ist. Wenn mann nun keine Ermäßigten Beitragsgruppen hat und trotzdem einen Familienverband haben möchte, müsste man die Beitragsgruppen doppelt anlegen. Alternativ könnte man es auch ermöglichen Mitgliedern mit normaler Beitragsgruppe einen Vollzahler zuzuweisen. Ob das Sinnvoll ist oder eher für mehr Verwirrung sorgt weiß ich nicht.

**Comment by dippeal** [comment]
Die Möglichkeit zu schaffen die Zahler zuzuweisen und von einem Mitglied abbuchen zu lassen halte ist für ganz nett. Wie die jeweiligen Vereine es dann nutzen ist ja ihnen überlassen. Wir z.B. haben auch mehrere Beitragsgruppen, auch wenn sich die Beiträge gleich sind.

**Comment by JohannMaierhofer** [comment]
Ich denke, dass man dies ganz unabhängig vom Familienverband machen könnte.
Statt bei bei Zahlung die Daten eines Mitglieds zu kopieren könnte man als dritte Möglichkeit eine Referenz auf den Zahler eintragen. Der muss auch nicht im Familienverband sein bzw. es muss gar keinen dafür geben. Der Zahler könnte jedes Mitglied sein. 
Die Eingabe eines alternativen Zahler wie heute würde weiter Sinn machen. Dann kann man auch einen Namen eingeben der kein Mitglied ist.

**Reply by dominformant** [reply]
Genau so hatte ich mir das vorgestellt, dass es entweder eine zusätzliche Option neben Barzahlung, Überweisung und SEPA-Lastschrift gibt oder aber man bei SEPA eine Referenz auf das SEPA-Mandat eines anderen Mitglieds setzen kann. Das ganze ist eigentlich komplementär zu den Familienbeiträgen, die legen die Beitragshöhe fest, während dieses Feature nur den Zahlungsweg vorgibt.

**Comment by JohannMaierhofer** [comment]
Die Idee ist doch nicht so toll. Dann haben wir ja wieder einen neue Art Beitragsfamilie.
Da ist die Idee mit der Option "Durch Vollzahler" beim Familienmitglied doch die bessere Idee. Dann hat man halt bei gleichem Beitrag eine Beitragsgruppe doppelt. Das sollte auch nicht das Problem sein.

**Reply by dominformant** [reply]
Das verstehe ich jetzt nicht ganz, der Sinn der Familienbeiträge ist doch eigentlich bei reuzierten Beiträgen in einem Familienverband nachverfolgen zu können, welche Mitglieder zusammen gehören, oder? Klar würde auch eine gemeinsame Abbuchung eine Art Verbindung zwischen Mitgliedern herstellen, aber ja nur in Bezug auf den Zahlungsweg und nicht auf die Beitragshöhe. Insbesondere würde die gemeinsame Abbuchung auch für Zusatzbeiträge gelten, während Familienbeiträge ja nur für die regulären Jahresbeiträge relevant sind.

**Reply by JohannMaierhofer** [reply]
Wenn man für die Bezahlung auf ein anderes Mitglied verweist hat man ja wieder eine Beziehung. Dann möchte man wie beim Familienverband irgendwo sehen welche Bezahl Beziehungen existieren. Und man muss dann wieder prüfen, dass ein Zahler nicht austreten kann wenn er noch für jemanden zahlt.
Das ist ja dann das gleiche wie der Familienverband.
Mit dem neuen Schalter kann man halt den Familienverband auch für etwas anderes missbrauchen und eben nur eine reine Bezahlbeziehung erstellen.
Die Bezahlbeziehung muss auch die Zusatzbeiträge einbeziehen weil man in diesem Fall beim Familienmitglied ja keine Kontodaten mehr eingeben muss.

**Reply by lenilsas** [reply]
Das sehe ich auch so. Der Familienverband stellt eine Abhängigkeit voneinander her. Man kann sie die für reduzierte Beiträge oder für das gemeinsame Lastschriftmandat verwenden. Nur müssen halt, wie schon gesagt, zwei Beitragsgruppen bestehen als Normal und als Angehöriger, da sehe ich noch keine andere gute Lösung.
Und ich denke, dass doch in den allermeisten Fällen nur innerhalb einer Familie für andere (Angehörige) der Beitrag gezahlt wird, von daher macht auch die Bezeichnung Familienbeitrag Sinn, oder seht ihr das anders?

**Reply by JohannMaierhofer** [reply]
Ich sehe das auch so.
Wenn wir jetzt den Schalter einbauen müssen wir darüber reden wie das implementiert werden soll. Es werden die Zahlungen zu einer Lastschrift zusammen gefasst. Das ist klar.
Aber wie schaut es mit den Sollbuchungen (Mitgliedskonto) und den Buchungen aus die der Abrechnungslauf erzeugt. Diese dürften nicht zusammen gefasst werden und müssten jeweils den Mitgliedern zugeordnet werden. Sie sollten ja wohl im jeweiligen Mitgliedskonto auftauchen.

Für mich wäre noch interessant wer rechtlich eigentlich die Spendenbescheinigungen erhalten darf. Jedes Mitglied eine oder nur der der gezahlt hat mit dem ganzen Betrag.

**Reply by dominformant** [reply]
Jetzt verstehe ich wie das gemeint ist, tatsächlich würde eine zusätzliche gemeinsame Abbuchung das ganze irgendwie doppeln. Der Fall, dass eine Person Mitglied eines Familienverbandes ist, die Abbuchung aber über ein anderes Vereinsmitglied ohne Bezug zu der Familie ist, ist zwar vorstellbar, aber eher unwahrscheinlich :laughing:

**Reply by dominformant** [reply]
Wobei, macht es das ganze nicht in Bezug auf den Familienverband deutlich komplizierter, wenn man abbilden muss, dass in manchen Familien nur ein Zahler für alle existiert und in anderen jedes Mitglied selbst zahlt? Der theoretische Fall, dass in einer Familie zwei Personen über den gleichen Zahler zahlen und eine Person unabhängig dürfte ja auch wieder sehr selten sein, oder?

**Comment by JohannMaierhofer** [comment]
Evtl. würde es dann Sinn machen im Familienbeitrag View beim Vollzahler und den Mitgliedern neben dem Namen auch die Beitragsgruppe und den Betrag anzuzeigen.

**Comment by lenilsas** [comment]
Ich habe jetzt den Zahlungsweg "durch Vollzahler" umgesetzt. #286
